src/backend/dx11/src/lib.rs

```diff
@@ -1484,11 +1484,11 @@ impl hal::command::RawCommandBuffer<Backend> for CommandBuffer {
                     | ChannelType::Sfloat
                     | ChannelType::Uscaled
                     | ChannelType::Sscaled
-                    | ChannelType::Srgb => ClearColor::Float(unsafe { raw_clear.float32 }),
+                    | ChannelType::Srgb => ClearColor::Sfloat(unsafe { raw_clear.float32 }),
 
-                    ChannelType::Uint => ClearColor::Uint(unsafe { raw_clear.uint32 }),
+                    ChannelType::Uint => ClearColor::Suint(unsafe { raw_clear.uint32 }),
```

Comment:
I don't think "Suint" makes sense :) Should be "Uint"

Reply:
So I guess the `S` stands for signed? I was under the impression it meant `Single` like Sfloat, Dfloat for double.